docs/api.rst

```diff
@@ -12608,7 +12608,7 @@ Managing custom accounting rules
 
   .. _accounting_rules_fields:
 
-  :reqjsonarr boolean taxable: ``true`` if the event should be considered as taxable
+  :reqjsonarr object taxable: ``true`` if the event should be considered as taxable. Allows to link the property to an accounting setting.
```

Comment:
How is an object ``true``? So can be boolean or object? How does it look when an object? Can't understand from the doc entry

Reply:
I forgot to update the response object. It is always an object and the `value` attribute is what is a boolean. Fixed it along a better comment